packages/location-management/src/components/LocationTree/tests/index.test.tsx

```diff
@@ -134,39 +136,18 @@ describe('location-management/src/components/LocationTree', () => {
     });
 
     treeNode = wrapper.find('.ant-tree-list-holder-inner');
-    expect(treeNode.children().length).toBeGreaterThan(treedata.length); // as per structure make sure the parent tree is expended i.e more child
-    wrapper.unmount();
-  });
-
-  it('should highlight selected tree item', async () => {
```

Comment:
are we not doing this anymore?

Reply:
moved it here: https://github.com/opensrp/web/blob/7bfa52a4d8b5822536ea3035c46bd9a705a6515c/packages/location-management/src/components/LocationUnitList/tests/index.test.tsx#L198